### DISCUSSION TITLE
Visualize counter in RepeatCount

### DISCUSSION THREAD

**Discussion by msarvestani** [bonsai-rx/bonsai#1362] [Q&A]
I'm using RepeatCount to repeat a set number of trials of my workflow. How can I visualize the current trial-number?

![RepeatCount](https://github.com/bonsai-rx/bonsai/assets/12864873/e454e757-b758-4f14-a1e3-3cc1e7acbc48)

**Comment by bruno-f-cruz** [comment]
Something like this?
![image](https://github.com/bonsai-rx/bonsai/assets/7049351/368dbb33-9dee-494a-a578-8aba961707d7)

**Comment by msarvestani** [comment]
That worked, thanks!
